notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
<p><code>TRAIN_BATCH_SIZE</code> and <code>TRAIN_EPOCHS</code> are not defined yet (defined in the next cell), and returns error here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Good catch! Done!</p>